src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppEcsFargate/Generated/Recipe.cs

```diff
@@ -47,7 +46,7 @@ public class Recipe : Construct
         public string AddTargetGroup { get; } = "AddTargetGroup";
 
         public Recipe(Construct scope, IRecipeProps<Configuration> props)
-            : base(scope, "GeneratedRecipeConstruct")
+            : base(scope, "Recipe")
```

Comment:
I decided to change the default name to something shorter because this name is often used as part of the resource names created like bucket names.

Reply:
There should be a comment explaining why the construct id is constant here. And changing it will impact the stack.

Reply:
Done